packages/hash/subgraph/src/types/element.ts

```diff
@@ -35,7 +35,7 @@ export type EntityTypeWithMetadata = Omit<
 export type { OntologyElementMetadata } from "@hashintel/hash-graph-client";
 
 /** Plain JSON value and object definitions */
-type JsonValue = string | number | boolean | JsonObject | JsonValue[];
+type JsonValue = null | string | number | boolean | JsonObject | JsonValue[];
```

Comment:
I really wonder why there isn't a built in `JsonValue` type in TS..

Reply:
Yep I have no idea 🙃